ament_black/ament_black/main.py

```diff
@@ -39,6 +39,12 @@ def main(argv=sys.argv[1:]):
         help="The files or directories to check. For directories files ending "
         "in %s will be considered." % ", ".join(["'.%s'" % e for e in extensions]),
     )
+    parser.add_argument(
+        '--config',
+        metavar='path',
+        default=None,
+        dest='config_file',
+        help='The config file')
```

Comment:
Perhaps extend with 
```python
    if not os.path.exists(args.config_file):
        print("Could not find config file '%s'" % args.config_file,
              file=sys.stderr)
        return 1
```

After the `if not files:` check below.

In line with `ament_clang_format`: https://github.com/ament/ament_lint/blob/master/ament_clang_format/ament_clang_format/main.py#L64